compiler/backend/ccgstmts.nim

```diff
@@ -763,7 +673,7 @@ proc bodyCanRaise(p: BProc; n: PNode): bool =
       if bodyCanRaise(p, n[i]): return true
     result = false
 
-proc genTryGoto(p: BProc; t: PNode; d: var TLoc) =
```

Comment:
Oh this is fantastic, we're not screwing around with `TLoc`s nearly as much! IIRC, there is a bunch of unfortunate AST mutation surrounding those that precluded moving towards a sealable/cacheable AST.